### PR TITLE
Back end setup

### PR DESCRIPTION
**Pull request by ushift-d**
- API tests
- Seed Database

### CODE REVIEW

File: database/index.js
Comment (TandyN) [inline database/index.js]: Maybe have a lowercase 'id' so `product_id: Number,`? To keep things consistent

File: database/index.js
Comment (TandyN) [inline database/index.js]: Put this entire model in a separate file?

File: seedScript.js
Comment (TandyN) [inline seedScript.js:23]: Make the partNumber actual numbers?

File: seedScript.js
Comment (TandyN) [inline seedScript.js]: Better function name.. like `randNum` ? Can also use an arrow function. This includes for the rest of the page.

File: seedScript.js
Comment (TandyN) [inline seedScript.js]: The use of `+=` here?

File: seedScript.js
Comment (TandyN) [inline seedScript.js]: Is there a reason why you just can't use mongoose to create these models and push the promises into an array? Then use Promise.all instead when you are seeding? That would remove the need of having the insertProducts function.

File: test/index.test.js
Comment (TandyN) [inline test/index.test.js:29]: I'm not sure if this test is necessary to have in master?

File: server/index.js
Comment (TandyN) [inline server/index.js:13]: Everything is so close together?

**Review by TandyN — CHANGES_REQUESTED** [review]
You've...
- Created a Product model
- Created a seeding script that seeds 100 products from id 1 - 100
- Added 1 route

I'm not sure if you need the test created for that route you've put in as I do not see the relevance in what the route is even doing at the moment. Unless you intend your server to send `'Working Path'` in production/master branch.

File: server/index.js
Comment (ushift-d) [reply server/index.js:13]: Yah esLint told me I had to.

File: test/index.test.js
Comment (ushift-d) [reply test/index.test.js:29]: This file isn't real, I have an update on another branch using axios testing my api routes and creating a mock server

File: seedScript.js
Comment (ushift-d) [reply seedScript.js:23]: They aren't just number they're letters and numbers mixed

File: seedScript.js
Comment (ushift-d) [reply seedScript.js]: Just preference, I don't like using promises, but I'll look into refactoring this later down the line.

File: database/models/product.js
Comment (TandyN) [inline database/models/product.js:18]: Approved after getting rid of the white space.

**Review by TandyN — COMMENTED** [review]
I see that you've moved the product model to a different file & corrected a function name.

File: server/index.js
Comment (TandyN) [reply server/index.js:13]: That is not what ESLint is telling me. May want to look at your config if you don't like this.

**Review by TandyN — APPROVED** [review]
Please remove the white space in database/models/products.js in your model before merging.

**Pull request merged**
